uint8 motor
uint8 direction
uint8 use_ramping
uint8 job_id
uint8 microstep

float32 rotations

uint32 pulse_interval
uint32 pulse_on_period
uint32 ramping_steps
---
uint8 response
